### ISSUE TITLE
Related table does not show anymore in detail view

### ISSUE THREAD

**Issue by preliot** [laravel/nova-issues#1638]
- Laravel Version: 5.8.18
- Nova Version: 2.0.5
- PHP Version: 7.2.10
- Operating System and Version: Windows 10
- Browser type and version: Chrome 74 (64 bits)

### Description:

Related tables do not show anymore in the detail view. Parent table is correctly loaded and then i can see an empty table with the loading dots, but data never shows. This used to work.

### Steps To Reproduce:

```php
  /**
     * Get the fields displayed by the resource.
     *
     * @param  \Illuminate\Http\Request  $request
     * @return array
     */
    public function fields(Request $request): array {

        return [
            //...
                Fields\HasMany::make('Product locales'), 'catalogProductLocales', CatalogProductLocales::class),
        
            //...
       ];
```

What am i missing? Or is it a real bug?

**Comment by pehbehbeh** [comment]
We are having the same problem on a Resource with five relations - but strangely, on some page loads it works... 🤔 No errors or bad requests.

**Comment by pehbehbeh** [comment]
I just found #1629 - so this may be a duplicate.

**Comment by FaridAghili** [comment]
Yes, it's already fixed and will be in 2.0.6

**Comment by preliot** [comment]
> Yes, it's already fixed and will be in 2.0.6

How do you know? Is there a roadmap which i'm unaware of?

**Comment by pehbehbeh** [comment]
It was stated 19 days ago by @davidhemphill in https://github.com/laravel/nova-issues/issues/1608#issuecomment-493686855 ...

**Comment by preliot** [comment]
This is now fixed in the released 2.0.6.
